src/pages/resultsView/enrichments/ExpressionEnrichmentsContainer.tsx

```diff
@@ -193,27 +215,209 @@ export default class ExpressionEnrichmentContainer extends React.Component<IExpr
         });
     }
 
+
+    @autobind
+    private getData() {
+        return getBoxPlotDownloadData(
+            this.boxPlotData.result!,
+            "Group",
+            this.clickedGeneHugo + ", " + this.props.selectedProfile.name,
+            {}
+        )
+    }
+
+    public readonly horzAxisData = remoteData({
```

Comment:
I think everything pertaining to the boxplot should be wrapped into another file and component, i.e. ExpressionEnrichmentsBoxPlot.tsx